aesara/tensor/var.py

```diff
@@ -142,8 +145,6 @@ def __div__(self, other):
         except (NotImplementedError, TypeError):
             return NotImplemented
 
-    __truediv__ = __div__
```

Comment:
I think MyPy is returning an error because `__truediv__` is defined 2 times here (see l.171). Deleting this was fixing the typing error and did not result in other errors.